src/App.tsx

```diff
@@ -1,10 +1,46 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Errors } from './components/errors/errors';
+import { Footer } from './components/footer/footer';
+import { ToDoList } from './components/todoList/todoList';
+import { Header } from './components/header/header';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [status, setStatus] = useState('all');
+
+  useEffect(() => {
+    getTodos()
+      .then(todosFromServer => {
+        switch (status) {
+          case 'all':
+            setTodos(todosFromServer);
+            break;
+          case 'active':
+            setTodos(todosFromServer.filter(todo => !todo.completed));
+            break;
+
+          case 'completed':
+            setTodos(todosFromServer.filter(todo => todo.completed));
+            break;
+
+          default:
+            setTodos(todosFromServer);
+        }
+      })
+      .catch(() => setErrorMessage('Unable to load todos'))
+      .finally(() => {
+        setTimeout(() => setErrorMessage(''), 3000);
+      });
```

Comment:
If you don't have an error so you don't need to reset it. You can create a handler that sets an error and then clean it in 3 seconds, it will help you to don't repeat it in the future. Something like this:
```typescript
const handleError = (errorMessage: string) => {
  setErrorMessage(errorMessage);

  setTimeout(() => {
    setErrorMessage('');
  }, 3000);
};
```